app/components/transliterable_field_group_component.rb

```diff
@@ -0,0 +1,29 @@
+class TransliterableFieldGroupComponent < BaseComponent
```

Comment:
@aduth @zachmargolis This is still very much WIP, but I'd like your feedback on this new component (including its Ruby, ERB, and TypeScript parts) before I move on to writing tests.

Reply:
as a one quick note, this file indents by 4 but we indent by 2

Reply:
Thanks, I'll be running `rubocop` before taking this out of draft status. May also adjust my default ruby settings to indent by 2.